src/components/helpers/dom_helpers.ts

```diff
@@ -87,3 +87,29 @@ export function isMacOS(): boolean {
 export function isCtrlKey(ev: KeyboardEvent | MouseEvent): boolean {
   return isMacOS() ? ev.metaKey : ev.ctrlKey;
 }
+
+export async function convertImageToPng(imageUrl: string): Promise<Blob | null> {
+  return await new Promise((resolve, reject) => {
+    const image = new Image();
+    image.src = imageUrl;
+    image.addEventListener("load", () => {
```

Comment:
probably doesn't matter in practice, but should the event listener be attached before setting `src` ? If somehow `image.src = imageUrl;` is instant and synchronous, the event handler would never be called

Reply:
I just couldn't make it work without that, i might have missed something...